src/schnetpack/transform/neighborlist.py

```diff
@@ -29,6 +31,37 @@ class CacheException(Exception):
     pass
 
 
+class NeighborlistWrapper(Transform):
```

Comment:
why is this needed? why not add a separate preprocessing layer?

Reply:
for the training procedure you are right: one could just use a separate preprocessing layer. However, in the scope of MD and structure relaxation, this wrapper makes it more convenient to use NBL postprocessing. For example, the AtomsConverter in its current state does not allow the usage of additional pre/postprocessing. One would either have to implement a postprocessing parameter there or keep it as is and use the nbl wrapper instead.